Source/WebCore/testing/Internals.h

```diff
@@ -337,7 +337,7 @@ class Internals final : public RefCounted<Internals>, private ContextDestruction
     void setAutofilled(HTMLInputElement&, bool enabled);
     void setAutoFilledAndViewable(HTMLInputElement&, bool enabled);
     void setAutoFilledAndObscured(HTMLInputElement&, bool enabled);
-    enum class AutoFillButtonType { None, Contacts, Credentials, StrongPassword, CreditCard };
+    enum class AutoFillButtonType { None, Contacts, Credentials, StrongPassword, CreditCard, Loading };
```

Comment:
I think you also want to add this to `Source/WebCore/testing/Internals.idl` and write some tests :)

Reply:
yup, Aditya is helping me do that now